### PR TITLE
[ci skip] Update README and RDOC_MAIN

### PR DESCRIPTION
**Pull request by ftovaro**
This is just an update of the README, more about styling than in content, is easier to read now

RDOC_MAIN.rdoc has a fix of broken links too.

Here are some screenshots of how it looks with the changes:

image

image

🚀

### CODE REVIEW

**Comment by ftovaro** [comment]
@yhirano55 Of course, I'm gonna update it right know and thank you for the `[ci skip]` advice.

**Comment by guilleiguaran** [comment]
Can you generate API docs locally and take a screenshot of how it looks with your changes?

File: railties/RDOC_MAIN.rdoc [hunk 1]
```diff
@@ -1,48 +1,55 @@
-== Welcome to \Rails
+= Welcome to \Rails
 
-\Rails is a web-application framework that includes everything needed to
+== What's Rails
```
Comment (yhirano55) [inline railties/RDOC_MAIN.rdoc]: Let's use `\Rails`. And please confirm generated pages after `bundle exec rake rdoc && open doc/rdoc/index.html`.

**Review by yhirano55 — APPROVED** [review]

**Comment by fxn** [comment]
Hey! Looks good.

While "Model Layer" is a term that deserves some style, I don't really think we should style "Rails" at the top. Could you please revise that detail?

**Comment by ftovaro** [comment]
@fxn Done, let me update the screen shot too

**Comment by fxn** [comment]
Awesome, thank you!

**Comment by ftovaro** [comment]
@fxn Hey, i just realized that I did the change of style of the word only in RDOC_MAIN and not in README, so I'm sorry for that. Also, I just figured out that the links in RDOC_MAIN wasn't pointing to files on the project but pages on http://edgeapi.rubyonrails.org/ so I restored the links that were before.

**Comment by fxn** [comment]
@ftovaro my fault, should have spotted it in the review. Yes, please, let's fix it in another PR.

**Comment by guilleiguaran** [comment]
@ftovaro I already fixed the style of the word in fb9ed7b0dff7dcf8a00e42cd45ec645211c04afc.

Please send a PR to fix the links in the RDOC file :blush:

**Pull request merged**